indra/llinventory/llparcel.cpp

```diff
@@ -947,128 +772,6 @@ const std::string& LLParcel::getActionString(LLParcel::EAction action)
     return PARCEL_ACTION_STRING[index];
 }
 
-BOOL LLParcel::isSaleTimerExpired(const U64& time)
-{
-    if (mSaleTimerExpires.getStarted() == FALSE)
-    {
-        return FALSE;
-    }
-    BOOL expired = mSaleTimerExpires.checkExpirationAndReset(0.0);
-    if (expired)
-    {
-        mSaleTimerExpires.stop();
-    }
-    return expired;
-}
-
-BOOL LLParcel::isMediaResetTimerExpired(const U64& time)
-{
-    if (mMediaResetTimer.getStarted() == FALSE)
-    {
-        return FALSE;
-    }
-    BOOL expired = mMediaResetTimer.checkExpirationAndReset(0.0);
-    if (expired)
-    {
-        mMediaResetTimer.stop();
-    }
-    return expired;
-}
-
-
-void LLParcel::startSale(const LLUUID& buyer_id, BOOL is_buyer_group)
-{
-	// TODO -- this and all Sale related methods need to move out of the LLParcel 
-	// base class and into server-side-only LLSimParcel class
```

Comment:
This **TODO** comment is relevant: the server-only logic in `LLParcel` was supposed to be removed a long time ago.